app/services/cfe_service.rb

```diff
@@ -25,7 +25,7 @@ def create_assessment_id(cfe_connection, session_data)
 
     def result(cfe_connection, cfe_estimate_id, cfe_session_data)
       cfe_connection.api_result(cfe_estimate_id).tap do |calculation_result|
-        calculation_result.level_of_help = cfe_session_data["level_of_help"]
+        calculation_result.level_of_help = cfe_session_data["level_of_help"] || "certificated"
```

Comment:
should we avoid doing this if the controlled flag is set? [I did that in another change, and marked it with a 'TODO' when the flag is removed]